lib/option.rb

```diff
@@ -43,8 +43,8 @@ def display_name
     ["burstable", "Shared CPU", true, true]
   ].map { |args| VmFamily.new(*args) }
 
-  IoLimits = Struct.new(:max_ios_per_sec, :max_read_mbytes_per_sec, :max_write_mbytes_per_sec)
-  NO_IO_LIMITS = IoLimits.new(nil, nil, nil).freeze
+  IoLimits = Struct.new(:max_read_mbytes_per_sec, :max_write_mbytes_per_sec)
+  NO_IO_LIMITS = IoLimits.new(nil, nil).freeze
```

Comment:
```suggestion
  NO_IO_LIMITS = IoLimits.new.freeze
```

Values are optional when initializing structs, and default to nil.  If you think IoLimits values should always be frozen, consider changing `Struct.new` to `Data.define`.  You can think of Data like a frozen Struct.  In that case, you'll need to keep the `nil` arguments, because Data subclasses require values for each member.